node/src/sub_lib/blockchain_bridge.rs

```diff
@@ -22,8 +23,8 @@ pub struct BlockchainBridgeConfig {
 #[derive(Clone, PartialEq, Eq)]
 pub struct BlockchainBridgeSubs {
     pub bind: Recipient<BindMessage>,
-    pub report_accounts_payable: Recipient<ReportAccountsPayable>,
-    pub request_balances_to_pay_payables: Recipient<RequestBalancesToPayPayables>,
+    pub report_accounts_payable: Recipient<OutcomingPaymentsInstructions>,
+    pub pps_for_blockchain_bridge: Recipient<PayablePaymentSetup>,
```

Comment:
This name could maybe be improved, since it's a field in the `BlockchainBridgeSubs`.